src/client/src/apps/MainRoute/widgets/spotTile/components/PriceButton/styled.tsx

```diff
@@ -40,6 +40,10 @@ export const TradeButton = styled.button<{ direction: Direction; priceAnnounced:
   cursor: pointer;
   border: none;
   outline: none;
+  width: 122px;
+  min-width: 122px;
+  height: 59px;
+  min-height: 59px;
```

Comment:
Do these fixed widths apply to all usages of the price button? It doesn't introduce any regressions anywhere else does it? Did you check responsive mode, i.e. if you resize the app?

Reply:
I checked responsive mode for web, but I will check the mobile mode now.
Also, I think @fsarda and I will create another branch to make these children components responsive

Reply:
> The `isAnalyticView` prop seems to be getting passed through a number of components, but only used at the bottom-level.
> Have you considered putting this into React Context, instead?

good suggestions, I will do it in react context